### ISSUE TITLE
Feature Request: Dry Run Option

### ISSUE THREAD

**Issue by allcaps** [adamchainz/djade#79]
### Description

First of all, thank you for creating Djade. It's an excellent tool!

I have a feature request inspired by the `flake8-black` plugin, which emits a `BLK100 Black would make changes` warning. It doesn't show specifics.

Would you consider adding a similar feature to Djade? For example, a `--dry-run` flag that doesn’t alter the files but exits with a status code and outputs a message like "Djade would make changes" when formatting differences are detected.

This functionality is very useful in CI/CD pipelines as a quality gate to detect formatting issues without modifying files.

**Comment by adamchainz** [comment]
Yes, we can add this. We will call it `--check` like `black --check`, `ruff --check`, etc.
